pkgs/stdenv/cross/default.nix

```diff
@@ -63,6 +63,8 @@ in lib.init bootStages ++ [
              # `tryEval` wouldn't catch, wrecking accessing previous stages
              # when there is a C compiler and everything should be fine.
              then throw "no C compiler provided for this platform"
+           else if crossSystem.isDarwin
```

Comment:
We might need this to be an option. I think we want there to be a way to also use gcc for cross on Darwin

Reply:
That's possible but it would require nixpkgs to support Darwin as a cross target with a non-Darwin build platform, and I don't know if that's the case? I wasn't able to build a cc toolchain for it when I tried, anyway.

Reply:
Oh yeah, forgot this was "crossSystem" , not "localSystem"

Reply:
Yeah - that's not supported. But we do want to support building statically on native Darwin gcc. That might not be as useful as I was originally thinking though.